engineyard/ey-cookbooks-dev-v7
# [FBZ-10398] - Migrate elasticsearch cookbook to stack v7

**Description of your patch**
Install and configure Elasticsearch by migrating recipe from v6 to v7

**Recommended Release Notes**
Added support for Elasticsearch installation 

**Estimated risk**
Medium

**Components involved**
ey-elasticsearch

**Dependencies**
none

**Description of testing done**

* Create a new environment with v7 as stack
* Uploaded custom-elasticsearch recipe as per process defined on https://github.com/engineyard/ey-cookbooks-stable-v7/tree/fbz10398_migrate_elasticsearch_recipe/custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch#elasticsearch 
* Added utility instance named `elasticsearch`.
* SSH into instance check for elasticsearch service running using `systemctl status elasticsearch.service` version installed  and verify version using `curl localhost:9200`. 

**QA Instructions**
* Create a new environment with v7 as stack
* upload `custom-elasticsearch` recipe as per process defined on https://github.com/engineyard/ey-cookbooks-stable-v7/tree/fbz10398_migrate_elasticsearch_recipe/custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch#elasticsearch 
* Add utility instance named `elasticsearch` and allow it to boot.
* SSH into instance check for elasticsearch service running using `systemctl status elasticsearch.service` version installed  and verify version using `curl localhost:9200`.
